base/ubi9/Dockerfile

```diff
@@ -39,83 +39,121 @@ RUN dnf -y reinstall shadow-utils && \
 # Download and install gh-cli depending on the architecture.
 # See release page for details https://github.com/cli/cli/releases/tag/v2.78.0
 RUN \
-    TEMP_DIR="$(mktemp -d)"; \
-    cd "${TEMP_DIR}"; \
-    GH_VERSION="2.78.0"; \
-    GH_ARCH="linux_$TARGETARCH"; \
-    GH_TGZ="gh_${GH_VERSION}_${GH_ARCH}.tar.gz"; \
-    GH_TGZ_URL="https://github.com/cli/cli/releases/download/v${GH_VERSION}/${GH_TGZ}"; \
-    GH_CHEKSUMS_URL="https://github.com/cli/cli/releases/download/v${GH_VERSION}/gh_${GH_VERSION}_checksums.txt"; \
-    curl -sSLO "${GH_TGZ_URL}"; \
-    curl -sSLO "${GH_CHEKSUMS_URL}"; \
-    sha256sum --ignore-missing -c "gh_${GH_VERSION}_checksums.txt" 2>&1 | grep OK; \
-    tar -zxv --no-same-owner -f "${GH_TGZ}"; \
-    mv "gh_${GH_VERSION}_${GH_ARCH}"/bin/gh /usr/local/bin/; \
-    mv "gh_${GH_VERSION}_${GH_ARCH}"/share/man/man1/* /usr/local/share/man/man1; \
-    cd -; \
-    rm -rf "${TEMP_DIR}"
+    TEMP_DIR="$(mktemp -d)" && cd "${TEMP_DIR}" && \
+    GH_VERSION="2.78.0" && \
+    case "$TARGETARCH" in \
+        amd64) GH_ARCH="linux_amd64" ;; \
+        arm64) GH_ARCH="linux_arm64" ;; \
+        ppc64le) GH_ARCH="linux_ppc64le" ;; \
+        *) echo "Unsupported architecture: $TARGETARCH" && exit 0 ;; \
```

Comment:
Could we follow this `case` pattern for the other `RUN` statements (`ripgrep`, `bat`, `fd`, `kubedock`) as well?